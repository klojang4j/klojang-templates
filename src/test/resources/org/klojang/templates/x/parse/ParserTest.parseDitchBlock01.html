<html>
<body>
<!--%%--><p>I must be ditched</p><!--%%-->
<p>I must not be ditched</p>
<!--%%-->
I must
also be
ditched
<!--%%-->~%foo%<!--%%--> Me Too!<!--%%--></body>
</html>