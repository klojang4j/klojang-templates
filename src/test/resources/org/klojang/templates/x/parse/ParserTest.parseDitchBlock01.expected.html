<html>
<body>

<p>I must not be ditched</p>
bar</body>
</html>